src/test/java/com/easyrest/tests/Listeners.java

```diff
@@ -8,19 +8,24 @@
 
 public class Listeners extends BaseTest implements ITestListener {
 
+    public void onTestStart(ITestResult result) {
+        test = extent.createTest(result.getTestClass().getName() + " : " +
+                result.getMethod().getMethodName());
+    }
+
    public void onTestSuccess(ITestResult result) {
-// TODO Auto-generated method stub
-        System.out.println("Success of test cases and its details are : "+result.getName());
+       test.pass(result.getMethod().getMethodName());
     }
 
     public void onTestFailure(ITestResult result) {
-       String testMethodName = result.getMethod().getMethodName();
+        test.fail(result.getMethod().getMethodName());
+        String testMethodName = result.getMethod().getMethodName();
         try {
             BaseTest instance = (BaseTest)result.getInstance();
             driver = instance.getDriver();
-        } catch (Exception e) {
+         } catch (Exception e) {
             System.out.println(e);
```

Comment:
You almost done, just one place left) It's up to you of course

Reply:
> Just one place left) It's up to you of course System.out.println(e);

fix it